docs/scanners/pattern_search.md

```diff
@@ -9,7 +9,7 @@ The scanner also allows the options below.  These options can be set globally an
 * `exclude_extension` and `include_extension` for excluding and including file extensions, respectively. While these options can be combined, exclusions take precedence when extensions conflict (are both included and excluded) in declarations.
 * `exclude_directory` for excluding directories whose name matches GLOB.  It appears that sift does not support `/`s in the directory name.
 * `exclude_filepaths` for excluding file paths. Note the file paths must be regular paths, not GLOB, and cannot include regular expressions.
-
+* `blocksize` for defining the amount of memory that will be used to limit the parser. By default this is set to 256 KB and can be increased based on the application needs
```

Comment:
Small copy suggestions:

Add "blocksize in bytes (with optional suffix K|M)"

"By default this is set to 256KB and can be increased" => "By default this is set to 256K and can be increased" to avoid confusion on the format of the default value.

```
* `blocksize` - (blocksize in bytes with optional suffix K|M) for defining the amount of memory that will be used to limit the parser.   By default this is set to 256K and can be increased based on the application needs

```